SampleCode/README.md

```diff
@@ -1,30 +1,33 @@
 # Azure AI Document Intelligence Code Samples
-This repository contains example code snippets showing how Azure AI Document Intelligence can be used to get insights from documents.
 
-# Samples for REST API
-These code samples show common scenarios for using the REST API for Azure AI Document Intelligence in python and Jupyter notebooks.
-| File Name | Description |
-| --- | --- |
-| sample_get_all_text_lines.ipynb and sample_get_all_text_lines.py | Get all text lines from a document |
-| sample_get_all_tables.ipynb and sample_get_all_tables.py | Get all tables from a document |
-| ...| ... |
+> [!NOTE]
+> Form Recognizer is now **Azure AI Document Intelligence**!
+
+This repository contains example code snippets showing how Azure AI Document Intelligence can be used to get insights from documents.
 
 # Samples for Python SDK
-Refer to [this repo](https://github.com/Azure/azure-sdk-for-python/tree/azure-ai-formrecognizer_3.3.0b1/sdk/formrecognizer/azure-ai-formrecognizer/samples#samples-for-client-library-versions-320-and-later) for the samples using the latest SDK version.
+Refer to [this repo](https://github.com/Azure/azure-sdk-for-python/tree/main/sdk/documentintelligence/azure-ai-documentintelligence/samples) for the samples using the [latest SDK version](https://learn.microsoft.com/python/api/overview/azure/ai-documentintelligence-readme?view=azure-python-preview&preserve-view=true).
 
 # Samples for .NET SDK
-Refer to [this repo](https://github.com/Azure/azure-sdk-for-net/blob/Azure.AI.FormRecognizer_4.1.0-beta.1/sdk/formrecognizer/Azure.AI.FormRecognizer/samples/README.md#common-scenarios-samples-for-client-library-version-400) for the samples using the latest SDK version.
+Refer to [this repo](https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/documentintelligence/Azure.AI.DocumentIntelligence) for the samples using the [latest SDK version](https://learn.microsoft.com/dotnet/api/azure.ai.documentintelligence.documentintelligenceadministrationclient?view=azure-dotnet-preview&preserve-view=true).
```

Comment:
This link points to a specific class. Should we correct it like this?
`https://learn.microsoft.com/dotnet/api/azure.ai.documentintelligence`

Reply:
removed